data/mods/ssb/random-teams.ts

```diff
@@ -871,14 +879,22 @@ export class RandomStaffBrosTeams extends RandomTeams {
 	randomStaffBrosTeam(options: {inBattle?: boolean} = {}) {
 		const team: PokemonSet[] = [];
 		const debug: string[] = []; // Set this to a list of SSB sets to override the normal pool for debugging.
-		const pool = debug.length ? debug : Object.keys(ssbSets);
+		// @ts-ignore
+		if (this.format.isAFD) {
+			for (let x = 0; x < 6; x++) {
+				debug.push('Fox');
+			}
```

Comment:
```suggestion
			debug = Array(6).fill('Fox');
```
I think this is a bit cleaner but either way is fine tbh

Reply:
debug is a const, so this crashes.

Reply:
...then make it mutable?

Reply:
i mean thats fair